### ISSUE TITLE
[Feature Request] Suppress logging start/stop messages

### ISSUE THREAD

**Issue by za3k** [kernc/logkeys#240]
I'd love to suppress `Logging started ...` and `Logging stopped at 2022-08-03 20:06:36+0100` as well as timestamps.

**Comment by q2dg** [comment]
Yes. This feature is necessary to not break structured log collection by tools like FluentBit, LogStash or similar
